schema/me/update_me_mutation.js

```diff
@@ -65,6 +65,14 @@ export default mutationWithClientMutationId({
       description: "The collector level for the user",
       type: GraphQLInt,
     },
+    price_range_min: {
+      description: "The minimum price collector has selected",
+      type: GraphQLInt,
+    },
+    price_range_max: {
+      description: "The maximum price collector has selected",
+      type: GraphQLInt,
+    },
```

Comment:
This could also be:

```graphql
price_range {
  min
  max
}
```

Reply:
Is this some kind of namespacing? Gravity would expect to receive the two params `price_range_min` and `price_range_max`. I don't immediately see how the above maps...?

Maybe you could elaborate on why the nested version would be better?

Reply:
Does the ‘namespace’ part explain it well enough? https://github.com/artsy/guides/blob/master/GraphQL-Schema-Design.md#how-to-model-our-graph

(Tbh in the case I don’t feel very strong about changing this to a more graph-like representation, was just taking the moment to tell you about that.)

Reply:
Ahh I see so the way you mentioned is more graph like. Got it. I'd like to get this merged so that I could continue to work on the reaction side. 

I don't really know how this might change the resolve function, but I wouldn't mind revisiting this to refactor and pairing with you to do so.

Reply:
👍